src/extensions/mountainview/MVSortingView/ViewContainerTabBar.tsx

```diff
@@ -8,9 +8,10 @@ type Props = {
     currentView: View | null
     onCurrentViewChanged: (v: View) => void
     onViewClosed: (v: View) => void
+    active: boolean
 }
 
-const ViewContainerTabBar: FunctionComponent<Props> = ({ views, currentView, onCurrentViewChanged, onViewClosed }) => {
+const ViewContainerTabBar: FunctionComponent<Props> = ({ views, currentView, onCurrentViewChanged, onViewClosed, active }) => {
```

Comment:
Style the tabs based on whether they are selected and whether they are in a group (north/south) that is active